include/seastar/websocket/server.hh

```diff
@@ -71,9 +207,15 @@ public:
         , _read_buf(_fd.input())
         , _write_buf(_fd.output())
     {
+        pipe<buff_t> pip{PIPE_SIZE};
```

Comment:
Regarding your `pipe` question raised at our sync: I think it's a fitting abstraction here. Originally I thought about using a `seastar::queue<temporary_buffer<char>>`, but that's pretty much what `pipe` is - it's literally implemented as a queue of temporary buffers. Looks good.